packages/cli-server-api/src/devToolsMiddleware.ts

```diff
@@ -47,18 +47,36 @@ function launchDevTools(
 function startCustomDebugger({
   watchFolders,
   customDebugger,
+  host,
+  port,
 }: {
   watchFolders: ReadonlyArray<string>;
   customDebugger: string;
+  host: string | undefined;
```

Comment:
Wouldn't it be better to do `host = 'localhost'` right in the method declaration? (line 50, I believe)

Reply:
Also, we already have similar code here https://github.com/react-native-community/cli/blob/c23e16192f8d23915c14a11e54c3d5ffdfb243df/packages/cli-server-api/src/devToolsMiddleware.ts#L16. 

Potential code smell - we have default value in two different places - what if they get out of sync in the future?

Recommendation: set default value of `host` in this method https://github.com/react-native-community/cli/blob/c23e16192f8d23915c14a11e54c3d5ffdfb243df/packages/cli-server-api/src/devToolsMiddleware.ts#L34 before passing it to any `launchXDebugger`

Reply:
Yeah, makes sense. I was thinking the same but didn't feel confident enough to change the code more then necessary.